scripts/aad-app-reg.sh

```diff
@@ -350,18 +423,21 @@ fi
 # This tag ensures the app is listed in "Enterprise applications"
 az ad sp update --id $spId --set tags="['WindowsAzureActiveDirectoryIntegratedApp']"
 
-# Grant admin consent on the required resource accesses (Graph API)
-if [[ $grantAdminConsent -eq 1 ]]; then
-    echo "Granting admin consent for ${appName} API app (service principal ID ${spId}) - NOTE: Directory admin privileges required for this step"
-    grant_admin_consent $spId $msGraphObjectId $directoryReadAllId
-    grant_admin_consent $spId $msGraphObjectId $userReadAllId
-fi
+# If a TRE core app reg
+if [[ $workspace -eq 0 ]]; then
+
+  # Grant admin consent on the required resource accesses (Graph API)
+  if [[ $grantAdminConsent -eq 1 ]]; then
+      echo "Granting admin consent for ${appName} API app (service principal ID ${spId}) - NOTE: Directory admin privileges required for this step"
+      grant_admin_consent $spId $msGraphObjectId $directoryReadAllId
+      grant_admin_consent $spId $msGraphObjectId $userReadAllId
+  fi
 
-# Now create the app for the Swagger UI
-declare scope_openid=$(get_msgraph_scope "openid")
-declare scope_offline_access=$(get_msgraph_scope "offline_access")
+  # Now create the app for the Swagger UI
```

Comment:
I think the indentation  went wrong here right?

Reply:
Hmm, think it is correct, its part of the block on 427 to 519